plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -315,6 +315,7 @@ export class EventsProcessor {
                 await this.db.createPerson(
                     DateTime.utc(),
                     {},
+                    {},
```

Comment:
nit: if we're likely to do a lot of this, then could be worth converting the argument to an object.

but not something to consider now, let's move fast here